server/request_sendrawtx.go

```diff
@@ -71,6 +71,12 @@ func (r *RpcRequest) handle_sendRawTransaction() {
 
 	// If users specify a bundle ID, cache this transaction
 	if cacheId := r.req.URL.Query().Get("bundle-id"); cacheId != "" {
+		if !r.allowTxCache {
```

Comment:
I wonder if this check should cause rpc calls to fail, since users can't even check if it's enabled

Reply:
yes it will make it fail. users wouldn't check because they are using metamask anyway to create bundles. don't think that failing really is an issue 🤔